src/entry.ts

```diff
@@ -190,7 +193,9 @@ class Entry {
                         value = this.getPropertySafely(value, path[index + 1]);
                     } else {
                         // block value without uid
-                        value = this.getPropertySafely(value, path[index + 1])[blockId];
+                        value = this.getPropertySafely(value, path[index + 1])[
```

Comment:
can you pls check the formatting ? @Amitkanswal

Reply:
done